_data/events_gdcr2019/London_Codurance.json

```diff
@@ -0,0 +1,32 @@
+{
+  "title": "GDCR19 in London (@Codurance)",
+  "url": "https://www.meetup.com/london-software-craftsmanship/events/265499324/",
+  "moderators": [
+      {
+          "name": "Christina Ripley-Gonzalez",
+          "url": "https://twitter.com/TrinkeG"
+      },
+      {
+          "name": "Jorge Gueorguiev Garcia",
+          "url": "https://twitter.com/yefoakira"
+      }
+  ],
+  "sponsors": [
+    {
+      "name": "Codurance",
+      "url": "codurance.com"
```

Comment:
Please, add the url protocol because now... :bomb: :sweat_smile:

Reply:
Ah, a test that needs to be added? ;-)

Corrected now